vignettes/cookbook_preparatory_steps.Rmd

```diff
@@ -18,106 +18,108 @@ library(pacta.loanbook)
 
 # Preparatory Steps
 
-This section provides an overview of the preparatory steps that need to be taken before running the PACTA for Banks analysis. It includes information on the required input data sets and the required software.
+This section outlines the preparations that are necessary before running the PACTA for Banks analysis, including sourcing the required input data sets and software.
 
 ## Required Input Data Sets
 
-The PACTA for Banks analysis requires a number of input data sets to run. Some of these can be obtained from external sources, while others need to be prepared by the user. Furthermore, some of the input data sets are optional. 
+The PACTA for Banks analysis requires several input data sets, some from external sources and others prepared by the user. Certain data sets are optional.
 
-The main input data sets required for the analysis are the following:
+The key required inputs include:
 
-### Asset-based Company Data (ABCD)
+### Asset-Based Company Data (ABCD)
 
-- required input
-- external source
+- Required input
```

Comment:
Rather nitpicky, but it seemed to make sense to have these start with capital letters (given they are the first thing you see at the head of each section